client/components/ManageBotRunDialog/RetryCancelledCollectionsButton/index.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button } from 'react-bootstrap';
+import { useMutation } from '@apollo/client';
+import { RETRY_CANCELLED_COLLECTIONS } from '../queries';
+
+const RetryCancelledCollectionsButton = ({
+    collectionJob,
+    onClick = () => {}
+}) => {
+    if (!collectionJob) {
+        return null;
+    }
+
+    const [retryCancelledCollections] = useMutation(
+        RETRY_CANCELLED_COLLECTIONS,
+        {
+            variables: {
+                collectionJobId: collectionJob.id
+            }
+        }
```

Comment:
Not important lol, but this could fit on one line, up to you if you prefer the aesthetics of that.

Reply:
Oh yeah I rarely fight back against prettier even when I am able to. Agree that it is better as one line.